hostinit/setup.go

```diff
@@ -546,17 +548,33 @@ func (init *HostInit) ProvisionHost(ctx context.Context, h *host.Host) error {
 			})
 			return nil
 		}
+		event.LogProvisionFailed(h.Id, output)
```

Comment:
Should we really LogProvisionFailed, which is user facing, if we're going to try again?

Reply:
This doesn't change the host status, and otherwise, we'll never log this data.

I'm pretty sure that we do not actually have any logic that depends on the event log.

Reply:
It does appear in the host log though, which could be confusing, maybe?